### PR TITLE
Update put version tests to check is_published flag

### PR DESCRIPTION
**Pull request by CarlHembrough**
### What

Update put version tests to check is_published flag. The is_published flag is added to the instance node in the graph DB on publish of the version

### How to review

Review changes / test against https://github.com/ONSdigital/dp-dataset-api/pull/153

### Who can review

Anyone

### CODE REVIEW

**Comment by mattrout92** [comment]
It wasn't me it was Nathan!

File: publishing/datasetAPI/put_instance_test.go [hunk 1]
```diff
@@ -139,13 +145,16 @@ func TestSuccessfullyPutInstance(t *testing.T) {
 					checkEditionDoc(datasetID, instanceID, edition.Next)
 
 					Convey("and the dataset_id, edition and version values are set a properties on the neo4j instance node", func() {
-						neoDatasetID, neoEdition, neoVersion, err := neo4j.GetInstanceProperties(cfg.Neo4jAddr, instanceID)
+
+						instanceProps, err := neo4JStore.GetInstanceProperties(instanceID)
 						if err != nil {
-							t.Error(err)
+							log.ErrorC("failed to get properties from neo4j instance node", err, nil)
```
Comment (nshumoogum) [inline publishing/datasetAPI/put_instance_test.go]: Think about using `t.Errorf(...)` like on line 112 and 175

File: publishing/datasetAPI/put_version_test.go [hunk 2]
```diff
@@ -20,21 +21,35 @@ import (
...
 
-	datasetID := uuid.NewV4().String()
-	editionID := uuid.NewV4().String()
-	instanceID := uuid.NewV4().String()
-
 	datasetAPI := httpexpect.New(t, cfg.DatasetAPIURL)
 
+	neo4JStore, err := neo4j.NewDatastore(cfg.Neo4jAddr, "", neo4j.GenericHierarchyCPIHTestData)
+	if err != nil {
+		log.ErrorC("unable to connect to neo4j", err, nil)
```
Comment (nshumoogum) [inline publishing/datasetAPI/put_version_test.go]: Again think about using `t.Errorf(...)` instead of logging error

**Review by nshumoogum — CHANGES_REQUESTED** [review]

File: publishing/datasetAPI/put_instance_test.go (hunk 1 above)
Comment (CarlHembrough) [reply publishing/datasetAPI/put_instance_test.go]: Done

File: publishing/datasetAPI/put_version_test.go (hunk 2 above)
Comment (CarlHembrough) [reply publishing/datasetAPI/put_version_test.go]: Done 👍

**Review by nshumoogum — APPROVED** [review]

**Pull request merged**